rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -423,12 +633,92 @@ namespace rocal{
         m.def("GetIntValue",&rocalGetIntValue);
         m.def("GetFloatValue",&rocalGetFloatValue);
         // rocal_api_data_transfer.h
-        m.def("rocalCopyToOutput",&wrapper_copy_to_output);
-        m.def("rocalCopyToOutputTensor",&wrapper_tensor);
-        m.def("rocalCopyToOutputTensor32",&wrapper_tensor32);
-        m.def("rocalCopyToOutputTensor16",&wrapper_tensor16);
-        m.def("rocalCopyCupyToOutputTensor32",&wrapper_copy_cupy_tensor32);
-        m.def("rocalCopyCupyToOutputTensor16",&wrapper_copy_cupy_tensor16);
+        m.def("getOutputTensors", [](RocalContext context) {
+            rocalTensorList * tl = rocalGetOutputTensors(context);
+            py::list list;
+            unsigned int size_of_tensor_list = tl->size();
+            for (uint i =0; i< size_of_tensor_list; i++)
+                list.append(tl->at(i));
+            return list; 
+        });
+        m.def("getBoundingBoxCount",&rocalGetBoundingBoxCount);
+        m.def("getImageLabels", [](RocalContext context) {
+            rocalTensorList *labels = rocalGetImageLabels(context);
+            return py::array(py::buffer_info(
+                            (int *)(labels->at(0)->buffer()),
+                            sizeof(int),
+                            py::format_descriptor<int>::format(),
+                            1,
+                            {labels->size()},
+                            {sizeof(int) }));
+        });
+        m.def("getBoundingBoxLabels", [](RocalContext context) {
+            rocalTensorList *labels = rocalGetBoundingBoxLabel(context);
+            py::list labels_list;
+            py::array_t<int> labels_array;
+            for (int i = 0; i < labels->size(); i++) {
```

Comment:
why are we doing this data conversion here?

Reply:
Each image can have more than one label. Hence this conversion